src/components/pages/Dashboard/ServiceMap.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { useState } from 'react';
+import ReactMapGL, { FlyToInterpolator } from 'react-map-gl';
```

Comment:
Would love to see some documentation on the reasons for using react-map-gl~~

Reply:
Great work! Thanks for taking the time to understand how to implement ReactMapGL into the project!